src/main/java/seedu/address/commons/core/Commands.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.commons.core;
+
+import seedu.address.logic.commands.AddCommand;
+import seedu.address.logic.commands.ClearCommand;
+import seedu.address.logic.commands.ClearHistoryCommand;
+import seedu.address.logic.commands.DeleteCommand;
+import seedu.address.logic.commands.EditCommand;
+import seedu.address.logic.commands.ExitCommand;
+import seedu.address.logic.commands.FindCommand;
+import seedu.address.logic.commands.HelpCommand;
+import seedu.address.logic.commands.HistoryCommand;
+import seedu.address.logic.commands.ListCommand;
+import seedu.address.logic.commands.RedoCommand;
+import seedu.address.logic.commands.SelectCommand;
+import seedu.address.logic.commands.UndoCommand;
```

Comment:
Might have to update this for the full list of commands.

Reply:
Yup. I would love to `import...*` but nope. And would also love to run some kind of `for` loop instead of manually going through the commands, but also nope. Oh well

Reply:
So this means that at the current moment, based on where this branch branched off, I won't be able to add in the new commands yet. I think I can open an issue for this, and periodically update the list though.